rover/src/main/java/io/rover/ui/BlockLayoutManager.java

```diff
@@ -51,7 +51,7 @@
     private BlockProvider mBlockProvider;
     private float density;
     private int verticalScrollOffset = 0;
-    private int bottomLimit = 0;
+    private Double totalLayoutHeight = 0.0;
```

Comment:
why not use the primitive type `double`?

Reply:
I wanted to make use of the integer coercion, which is (apparently) only available on the Double boxed type.